StashSearch/TraderScreenComponent.cs

```diff
@@ -316,5 +341,27 @@ private bool CheckTradingTableEmpty()
 
             return true;
         }
+
+        private void PopulateAutoComplete(AutoCompleteComponent autoComplete, StashGridClass grid, SearchController searchController, ref DateTime lastFill)
+        {
+            // don't populate if searching
+            if (searchController.IsSearchedState)
+            {
+                return;
+            }
+
+            // throttle this calculation
+            var timeDiff = DateTime.Now - lastFill;
+            if (timeDiff <= _autoCompleteThrottleTime)
+            {
+                return;
+            }
+            lastFill = DateTime.Now;
```

Comment:
While this is a super rare issue, it's worth mentioning: it's more preferable to use `DateTime.UtcNow` over `DateTime.Now` since `DateTime.Now` can be affected by daylight savings transitions.

See [this](https://rules.sonarsource.com/csharp/RSPEC-6561/) for more details.

Reply:
That's definitely true, I had forgotten that `DateTime.Now` was localized. Will change.